packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -73,6 +73,16 @@ Sandia National Laboratories, Albuquerque, NM, USA
 //#define TACHO_TEST_LEVELSET_TOOLS_KERNEL_OVERHEAD
 //#define TACHO_ENABLE_LEVELSET_TOOLS_USE_LIGHT_KERNEL
 
+#if (defined(KOKKOS_ENABLE_CUDA) && defined(TACHO_HAVE_CUSPARSE))
+  #if (CUSPARSE_VERSION >= 11400)
+  #define TACHO_CUSPARSE_SPMV_ALG CUSPARSE_SPMV_ALG_DEFAULT
+  #define TACHO_CUSPARSE_SPMM_ALG CUSPARSE_SPMM_ALG_DEFAULT
+  #else
+  #define TACHO_CUSPARSE_SPMV_ALG CUSPARSE_MV_ALG_DEFAULT
+  #define TACHO_CUSPARSE_SPMM_ALG CUSPARSE_MM_ALG_DEFAULT
+  #endif
+#endif
```

Comment:
This will work, but ``CUSPARSE_MM_ALG_DEFAULT`` was deprecated starting with ``CUSPARSE_VERSION == 11000`` so this would cause warnings if ``11000 <= CUSPARSE_VERSION < 11400``. Here's what we use in KokkosKernels:

```suggestion
#if (defined(KOKKOS_ENABLE_CUDA) && defined(TACHO_HAVE_CUSPARSE))
  #if (CUSPARSE_VERSION >= 11400)
    #define TACHO_CUSPARSE_SPMV_ALG CUSPARSE_SPMV_ALG_DEFAULT
  #else
    #define TACHO_CUSPARSE_SPMV_ALG CUSPARSE_MV_ALG_DEFAULT
  #endif
  #if (CUSPARSE_VERSION >= 11000)
    #define TACHO_CUSPARSE_SPMM_ALG CUSPARSE_SPMM_ALG_DEFAULT
  #else
    #define TACHO_CUSPARSE_SPMM_ALG CUSPARSE_MM_ALG_DEFAULT
  #endif
#endif
```

Reply:
Thank you, @brian-kelley. I will make the change!! I am trying to fix my commit history.